start.php

```diff
@@ -75,16 +75,7 @@ function registration_randomizer_tarpit($wait = 5) {
 	elgg_set_plugin_setting($setting_name, $count + 1, 'registration_randomizer');
 	registration_randomizer_log("Tarpitting $ip for $wait seconds after $count failures.", false);
 
-	if ($wait > 0) {
-		// close mysql connections for the time of a sleep
-		mysql_close(_elgg_services()->db->getLink('read'));
```

Comment:
Look at the issue that commit fixes or refs.

On Tue, Aug 4, 2015 at 10:28 PM Evan Winslow <notifications@github.com>
wrote:

> In start.php
> <https://github.com/brettp/elgg-registration-randomizer/pull/7#discussion_r36264778>
> :
>
> > @@ -75,16 +75,7 @@ function registration_randomizer_tarpit($wait = 5) {
> >  	elgg_set_plugin_setting($setting_name, $count + 1, 'registration_randomizer');
> >  	registration_randomizer_log("Tarpitting $ip for $wait seconds after $count failures.", false);
> >
> > -	if ($wait > 0) {
> > -		// close mysql connections for the time of a sleep
> > -		mysql_close(_elgg_services()->db->getLink('read'));
>
> I don't see that in #2
> <https://github.com/brettp/elgg-registration-randomizer/pull/2>. Can you
> clarify?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/brettp/elgg-registration-randomizer/pull/7/files#r36264778>
> .
>